<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>get_radar_config</class>
 <widget class="QDialog" name="get_radar_config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Radar State</string>
  </property>
  <property name="windowIcon">
   <iconset resource="terminal.qrc">
    <normaloff>:/images/get_radar_config.png</normaloff>:/images/get_radar_config.png</iconset>
  </property>
  <widget class="QPushButton" name="OkButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>40</y>
     <width>75</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>OutputTypeCfg</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>59</y>
     <width>69</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SendQalityCfg</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>78</y>
     <width>77</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SendExtInfoCfg</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>97</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MotionRxState</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>116</y>
     <width>73</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RCS_Threshold</string>
   </property>
  </widget>
  <widget class="QLabel" name="read_status">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>21</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="write_status">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="distanse">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>59</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="persistent_err">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>78</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="interference">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>97</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="temp_err">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>116</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="temporary_err">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>135</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="voltage_err">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>154</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sensor_id">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>173</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sort_index">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>192</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="power_cfg">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>211</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="relay_cfg">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="output_type">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="send_qality">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="send_ext">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="motion_state">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="rcs_threshold">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>m</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>21</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CtrlRelayCfg</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>21</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>NMWReadStatus</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>40</y>
     <width>82</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>NMWWriteStatus</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>59</y>
     <width>78</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MaxDistanseCfg</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>78</y>
     <width>78</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Persistent_Error</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>97</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Interference</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>116</y>
     <width>92</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Temperature_Error</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>135</y>
     <width>82</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Temporary_Error</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>154</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltage_Error</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>173</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sensor_ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>192</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SortIndex</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>211</y>
     <width>76</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>RadarPowerCfg</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>OkButton</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>label_15</zorder>
  <zorder>label_16</zorder>
  <zorder>label_17</zorder>
  <zorder>read_status</zorder>
  <zorder>write_status</zorder>
  <zorder>distanse</zorder>
  <zorder>persistent_err</zorder>
  <zorder>interference</zorder>
  <zorder>temp_err</zorder>
  <zorder>temporary_err</zorder>
  <zorder>voltage_err</zorder>
  <zorder>sensor_id</zorder>
  <zorder>sort_index</zorder>
  <zorder>power_cfg</zorder>
  <zorder>relay_cfg</zorder>
  <zorder>output_type</zorder>
  <zorder>send_qality</zorder>
  <zorder>send_ext</zorder>
  <zorder>motion_state</zorder>
  <zorder>rcs_threshold</zorder>
  <zorder>label_18</zorder>
  <zorder>label_12</zorder>
 </widget>
 <resources>
  <include location="terminal.qrc"/>
 </resources>
 <connections/>
</ui>
